charts/apisix-ingress-controller/crds/apisixic-crds.yaml

```diff
@@ -0,0 +1,3212 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.17.2
+  name: apisixconsumers.apisix.apache.org
+spec:
+  group: apisix.apache.org
+  names:
+    kind: ApisixConsumer
+    listKind: ApisixConsumerList
+    plural: apisixconsumers
```

Comment:
Missing shortnames.
This CRD is not the latest.

Reply:
Sorry. This PR was raised earlier and I didn’t notice that CRDs were updated. Now I have synchronized CRDs to the latest version.